src/Components/v2/Carousel.tsx

```diff
@@ -17,12 +19,32 @@ interface Props {
   renderLeftArrow?: Arrow
   renderRightArrow?: Arrow
   onArrowClick?: () => void
+  lastSlideVisible?: boolean
```

Comment:
Do these need to exist as props? It doesn't look like these wind up needing to be specified by a client. If it's just being used internally in this component, it prob doesn't need to be available as a prop.

Reply:
It looks like to me that they're the `Props` of `LargeCarousel` and the `State` of `Carousel`